src/relayclient/RelayProvider.ts

```diff
@@ -147,6 +147,9 @@ export default class RelayProvider implements HttpProvider {
   }
 
   _useGSN (payload: JsonRpcPayload): boolean {
+    if (payload.params[0] === undefined) {
+      return false
+    }
```

Comment:
When is it defined and when is it undefined?

Reply:
When running `web3.eth.getAccounts()` for example it was undefined. Probably defined on any call with params...